testing/run-write-tests.rst

```diff
@@ -134,6 +134,8 @@ When you are adding tests to an existing test file, it is also recommended
 that you study the other tests in that file; it will teach you which precautions
 you have to take to make your tests robust and portable.
 
+For tests of the C API, see Tests sections in :ref:`c-api`.
```

Comment:
Add an anchor for the Tests section, and add a `:ref:` to it here.

Reply:
I would, but there are 3 of them. IMO in this case it's better for readers to find the one they want by name in the TOC.

Reply:
Agreed.  I was going to suggest the same, but as @encukou said there are 3 sections and they are all easily discoverable in the `c-api` page.